pom.xml

```diff
@@ -16,15 +16,13 @@
 
     <groupId>org.eclipse.sw360.antenna</groupId>
     <artifactId>antenna-management</artifactId>
-    <version>${revision}${qualifier}</version>
+    <version>${revision}</version>
     <packaging>pom</packaging>
     <name>Antenna</name>
 
     <properties>
         <!-- >This is the main version applied throughout the project. Change this to increase all versions<-->
-        <revision>1.0.0</revision>
-        <!-- >The qualifier can be set to any value and will set the corresponding qualifier in all projects<-->
-        <qualifier>-SNAPSHOT</qualifier>
+        <revision>1.0.0-SNAPSHOT</revision>
```

Comment:
could we then not just not use a properties variable at all and instead use project.version?